client/templates/home/home.js

```diff
@@ -1,13 +1,13 @@
 Template.home.events({
-  "submit #productsExample": function(event, template){
-    var limit = $('#productsLimit').val();
+  "submit #searchingCode": function(event, template){
+    var searchedCode = $('#searchedCode').val();
 
-    if (!limit) {
-      alert('Enter the number of products you want to be published!');
+    if (!searchedCode) {
+      alert('Enter an item\'s look up code!');
       return;
     }
 
-    FlowRouter.go('/ProductsExample?limit=' + limit.trim());
+    FlowRouter.go('/SelectedProduct?searchedCode=' + searchedCode);
```

Comment:
We might want to use a regular param in the url here rather than a query param. I think query params are more generally used for things like the google or atmospherjs search pages where it shoes a list. In atmosphere, they use a query param when you are typing in the search and they are displaying the list, then when you choose a package, it uses a regular param in the URL. If we change it, it would be ```'/SelectedProduct/' + searchedCode``` instead of using the ```?``` with a parameter after that. You will also modify the declaration of the route to be ```/SelectedProduct/:searchedCode``` and that declares the name for the ```searchedCode``` param. To get this for your subscription you call ```FlowRouter.getParam``` instead of ```FlowRouter.getQueryParam```